src/game.cpp

```diff
@@ -5742,6 +5758,13 @@ int game::assign_faction_id()
     return ret;
 }
 
+long game::assign_creature_id()
+{
+    long ret = next_creature_id;
```

Comment:
We issue ~ 2 billion ids and then just give up on them being unique?  The likelihood of a collision is fairly infinitesimal, but it should at least throw a debug message when that line is crossed.